.github/workflows/test.yml

```diff
@@ -179,7 +179,7 @@ jobs:
             --platform linux/arm/v7 \
             -v `pwd`:/rdk \
             ghcr.io/viamrobotics/rdk-devenv:armhf-cache \
-            sudo -Hu testbot bash -lc 'sudo apt-get install -y python3-venv && cd /rdk && go test -v -tags=no_tflite ./...'
+            sudo -Hu testbot bash -lc 'sudo apt-get install -y python3-venv && cd /rdk && go test -v ./...'
```

Comment:
@abe-winter I assume it's all right to remove all instances of the no_tflite tag from the RDK now?

Reply:
yes OR hear me out we can put the no_tflite flag on every file to commemorate this change